src/styles/blocks/page-cards.scss

```diff
@@ -0,0 +1,19 @@
+.page-cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: PageWidth($card-width, $card-margin-right, $cards-per-row);
```

Comment:
Here you got the wrong value of width because you needn't use the margin-right value of last element in row.

Reply:
@bohdan-mykhailenko what u mean wrong? can't see the mistake, pls advise.